simperby/src/lib.rs

```diff
@@ -10,6 +10,8 @@ use simperby_network::AuthorizedNetwork;
 use thiserror::Error;
 use vetomint::Round;
 
+pub const PROTOCOL_VERSION: &str = "0.0.0";
```

Comment:
Is this going to be updated manually?

Reply:
Yes. It's protocol-specific information (so must reside in the code)